src/main/java/team/creative/creativecore/common/util/type/list/SortingList.java

```diff
@@ -192,7 +190,7 @@ public boolean contains(Object arg0) {
     
     @Override
     public boolean containsAll(Collection<?> arg0) {
-        return entries.containsAll(arg0);
+        return new HashSet<>(entries).containsAll(arg0);
```

Comment:
I don't see why you would do that?

Reply:
HashSet is more faster to run containsAll()

Reply:
I think it can be enhanced to use it as storage and not scare the GC every containsAll call. L

Reply:
I don't think this is faster ... you create a new HashSet every time you call contains all. This means all entries need to be hashed and put into buckets. If you have performance issues at least cash the hashset, then performance might be improved.

Reply:
reverted, i want to do some benchsmarks arround Sets and arrays